test/uk/gov/hmrc/tai/connectors/TaxAccountConnectorSpec.scala

```diff
@@ -103,7 +103,7 @@ class TaxAccountConnectorSpec extends PlaySpec with WireMockHelper with MockitoS
         val featureTogglesConfig = mock[FeatureTogglesConfig]
         val url = {
           val path = new URL(taxAccountUrlConfig.taxAccountUrlDes(nino, taxYear))
-          s"${path.getPath}?${path.getQuery}"
+          s"${path.getPath}"
```

Comment:
Since were're not doing any interpolation anymore, this could just be `path.getPath`

Reply:
updated the code

Reply:
the current URL to check seems doesn't working from around 11:30 am, can't confirm or deny if contains the same data structure

Reply:
@OktawianC are you sure you updated the code here as you are still doing string interpolation